plugins/askrene/layer.c

```diff
@@ -928,6 +969,15 @@ size_t layer_trim_constraints(struct layer *layer, u64 cutoff)
 		}
 	}
 
+	for (bias = bias_hash_first(layer->biases, &biasit); bias;
+	     bias = bias_hash_next(layer->biases, &biasit)) {
+		if (bias->timestamp < cutoff) {
+			bias_hash_delval(layer->biases, &biasit);
+			tal_free(bias);
+			num_removed++;
+		}
+	}
+
```

Comment:
This is cool. *but* 

1. askrene-age needs to document this new behaviour!
2. Changelog-None should be Changelog-Added: Plugins: `askrene` channel biases now have an associated timestamp, and are timed out by `askrene-age`.